source/_integrations/evse_wifi.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: EVSE WiFi
+description: Instructions on how to integrate a WiFi-equipped Simple EVSE Charging station with Home Assistant
+ha_category:
+  - Car
+ha_release: 0.38
+ha_iot_class: Local Polling
+ha_domain: evse_wifi
+ha_platforms:
+  - sensor
+  - switch
+  - button
+  - number
```

Comment:
Initial additions / PRs to Home Assistant (including the actual integration), may only contain a single platform at the time. The initial documentation PR this cannot contain multiple platforms either.